content/automate/ManageIQ/Infrastructure/VM/Migrate/StateMachines/VMMigrate.class/__methods__/update_migration_status.rb

```diff
@@ -1,18 +1,66 @@
 #
 # Description: This method updates the migration status
 #
+module ManageIQ
+  module Automate
+    module Infrastructure
+      module VM
+        module Migrate
+          module StateMachines
+            class UpdateMigrationStatus
+              def initialize(handle = $evm)
+                @handle = handle
+              end
 
-miq_task = $evm.root['vm_migrate_task']
-status   = $evm.inputs['status']
+              def main
+                @handle.log(:info, "Starting update_migration_status")
```

Comment:
@billfitzgerald0120 Can we remove the starting log message? The Automate engine creates a method starting log message.